custom-css.json

```diff
@@ -312,49 +522,723 @@
     "box-ordinal-group": {},
     "box-orient": {},
     "box-pack": {},
+    "box-shadow": {
+      "__additional_values": {
+        "inset": "inset 5em 1em gold",
+        "multiple_shadows": "3px 3px red, -6px -6px olive",
+        "spread_radius": "12px 12px 2px 1px teal"
+      }
+    },
+    "box-sizing": {
+      "__values": ["padding-box"]
+    },
+    "break-after": {
+      "__values": [
+        "always",
+        "avoid-column",
+        "avoid-page",
+        "column",
+        "recto",
+        "verso"
+      ]
+    },
+    "break-before": {
+      "__values": [
+        "always",
+        "avoid-column",
+        "avoid-page",
+        "column",
+        "recto",
+        "verso"
+      ]
+    },
     "buffered-rendering": {},
+    "caption-side": {
+      "__additional_values": {
+        "non_standard_values": [
+          "left",
+          "right",
+          "top-outside",
+          "bottom-outside"
+        ]
+      }
+    },
+    "clear": {
+      "__additional_values": {
+        "flow_relative_values": ["inline-end", "inline-start"]
+      }
+    },
+    "clip-path": {
+      "__additional_values": {
+        "-webkit-path": "-webkit-path('M20,20 L50,80 L80,20')",
+        "basic_shape": "circle(50px at 0 100px)",
+        "fill_and_stroke_box": ["fill-box", "stroke-box"],
+        "path": "path('M20,20 L50,80 L80,20')"
+      }
+    },
     "color-profile": {},
     "color-rendering": {},
+    "color-scheme": {
+      "__values": ["only", "only_dark", "only_light"]
+    },
+    "column-fill": {
+      "__values": ["balance-all"]
+    },
     "column-progression": {},
-    "custom-property": {},
```

Comment:
This removed a custom test by accident, restored in https://github.com/foolip/mdn-bcd-collector/pull/1945.